docs/src/gs/create-gpu-workload-cluster.md

```diff
@@ -0,0 +1,335 @@
+# Create a GPU workload cluster
+
+## Accessing GPU Shapes
+
+Some shapes are limited to specific regions and specific Availability Domains (AD).
+In order to make sure the workload cluster comes up check the region and AD for 
+shape availability.
+
+### Check shape availability
+
+Make sure the [OCI CLI][install-oci-cli] is installed. Then set the AD information if using 
+muti-AD regions.
+
+> NOTE: Use the [OCI Regions and Availability Domains][regions] page to figure out which 
+regions have multiple ADs.
+
+```bash
+oci iam availability-domain list --compartment-id=<your compartment> --region=<region>
+```
+
+Using the AD `name` from the output start searching for GPU shape availability.
+
+```bash
+oci compute shape list --compartment-id=<your compartment> --profile=DEFAULT --region=us-ashburn-1 --availability-domain=<your AD ID> | grep GPU
+ 
+"shape-name": "BM.GPU3.8"
+"shape-name": "BM.GPU4.8"
+"shape-name": "VM.GPU3.1"
+"shape": "VM.GPU2.1"
+```
+
+> NOTE: If the output is empty then the compartment for that region/AD doesn't have GPU shapes.
+If you are unable to locate any shapes you may need to submit a
+[service limit increase request][compute-service-limit]
+
+## Create a new GPU workload cluster an Ubuntu custom image
+
+> NOTE: Nvidia GPU drivers aren't supported for Oracle Linux at this time. Ubuntu is currently
+the only supported OS.
+
+When launching a multi-AD region shapes are likely be limited to a specific AD (example: `US-ASHBURN-AD-2`).
+To make sure the cluster comes up without issue specifically target just that AD for the GPU worker nodes.
+To do that modify the released version of the `cluster-template-failure-domain-spread.yaml` template.
+
+Make sure the modified template has only the `MachineDeployment` section(s) where there is GPU
+availability and remove all the others. See [the full example file][example-yaml-file]
+that targets only AD 2 (OCI calls them Availability Domains while Cluster-API calls them Failure Domains).
+
+### Virtual instances 
+
+The following command will create a workload cluster comprising a single 
+control plane node and single GPU worker node using the default values as specified in the preceding 
+[Workload Cluster Parameters][workload-cluster-parameters] table:
+
+> NOTE: The `OCI_NODE_MACHINE_TYPE_OCPUS` must match the OPCU count of the GPU shape.
+See the [Compute Shapes][compute-shapes] page to get the OCPU count for the specific shape.
+
+```bash
+OCI_COMPARTMENT_ID=<compartment-id> \
+OCI_IMAGE_ID=<ubuntu-custom-image-id> \
+OCI_SSH_KEY=<ssh-key>  \
+NODE_MACHINE_COUNT=1 \
+OCI_NODE_MACHINE_TYPE=VM.GPU3.1 \
+OCI_NODE_MACHINE_TYPE_OCPUS=6 \
+OCI_CONTROL_PLANE_MACHINE_TYPE_OCPUS=1 \
+OCI_CONTROL_PLANE_MACHINE_TYPE=VM.Standard3.Flex \
+CONTROL_PLANE_MACHINE_COUNT=1 \
+OCI_SHAPE_MEMORY_IN_GBS= \
+KUBERNETES_VERSION=v1.24.4 \
+clusterctl generate cluster <cluster-name> \
+--target-namespace default \
+--from cluster-template-failure-domain-spread.yaml | kubectl apply -f -
```

Comment:
can we call the file name example-yaml or something? may be it is better to say copy the contents, create a file called cluster-template-gpu.yaml and then use that term ?

Reply:
fixed